assets/js/views/ChargingSessions.vue

```diff
@@ -228,6 +226,16 @@ export default {
 			const modal = Modal.getOrCreateInstance(document.getElementById("sessionDetailsModal"));
 			modal.show();
 		},
+		csvHrefLink(locale, month) {
+			var url = "./api/sessions?format=csv&lang=" + locale;
```

Comment:
```suggestion
			var url = `./api/sessions?format=csv&lang=${this.$i18n.locale}`;
```
Dann kannst du den Param aus dem Template entfernen.

Reply:
Danach habe ich gesucht 😅